source/mongodb-handshake/handshake.rst

```diff
@@ -3,14 +3,14 @@ MongoDB Handshake
 =================
 
 :Spec Title: MongoDB Handshake
-:Spec Version: 1.2.0
+:Spec Version: 1.3.0
 :Author: Hannes Magnusson
 :Kernel Advisory: Mark Benvenuto
 :Driver Advisory: Anna Herlihy, Justin Lee
 :Status: Approved
 :Type: Standards
 :Minimum Server Version: 3.4
-:Last Modified: 2020-02-12
+:Last Modified: 2021-04-27
```

Comment:
That's a clearer indication than mixing it into the specification section. I'll add the terms section as suggested.

I'm going to remove the part about hello being supported in 5.0+ because it was backported to late point releases of 3.6+. It is not usable for initial handshake until 5.0 (because we require server API version support to know that hello is supported), but can be used for heartbeats if helloOk is returned from isMaster. Because helloOk was only backported to 4.2 and 4.4, but backports declined to 3.6 and 4.0 (see https://jira.mongodb.org/browse/SERVER-50412), there is no way for drivers to use the hello command on 3.6 and 4.0 even if the patch release supports it.

Reply:
Looking at the issues, I didn't find an indication why the backports to 3.6 and 4.0 were rejected. Do we have more info on that? I'm wondering if the wording change had anything to do with it, and whether we could get `helloOk` backported to 3.6 and 4.0 to complement the existence of the `hello` command.

Reply:
The terms section looks great.

@alcaeus the backport tickets linked in [SERVER-50412](https://jira.mongodb.org/browse/SERVER-50412) includes rationale for why the server team chose not to.